container/docker/factory.go

```diff
@@ -197,6 +208,80 @@ func startThinPoolWatcher(dockerInfo *dockertypes.Info) (*devicemapper.ThinPoolW
 	return thinPoolWatcher, nil
 }
 
+func getKernelVersion() (string, error) {
```

Comment:
Ok cool, thanks!
On Thu, Aug 4, 2016 at 1:30 PM Tim St. Clair <notifications@github.com>
wrote:

> In container/docker/factory.go
> <https://github.com/google/cadvisor/pull/1411#discussion_r73566520>:
>
> > @@ -197,6 +208,80 @@ func startThinPoolWatcher(dockerInfo *dockertypes.Info) (*devicemapper.ThinPoolW
> >  	return thinPoolWatcher, nil
> >  }
> >
> > +func getKernelVersion() (string, error) {
>
> This function already exists here:
> https://github.com/google/cadvisor/blob/master/machine/info.go#L142
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/cadvisor/pull/1411/files/d1e20410961d80cf2bde3a8a68c6249cf7c06c95#r73566520>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAABYme1PxGA4dN5oKe4pj-zPO7GPq7Eks5qciHAgaJpZM4Jc2Va>
> .
>